test/ConstExtraction/ExtractLiterals.swift

```diff
@@ -84,6 +84,119 @@
 // CHECK-NEXT:        "value": "\"Hello, World\""
 // CHECK-NEXT:      }
 // CHECK-NEXT:    ]
+// CHECK-NEXT:  },
+// CHECK-NEXT:  {
+// CHECK-NEXT:    "typeName": "ExtractLiterals.PropertyWrappers",
+// CHECK-NEXT:    "kind": "struct",
+// CHECK-NEXT:    "properties": [
+// CHECK-NEXT:      {
+// CHECK-NEXT:        "label": "_propertyWrapper1",
```

Comment:
This isn't necessarily a comment on this, and the current output actually captures what we do to the AST to de-sugar property-wrapped properties. 

But I wonder if its best to leave this as-is and let clients figure out how to connect a property with an attribute to its corresponding underlying `_` and `$` properties, or if we can bundle them up here somehow. Wdyt @jPaolantonio?

Reply:
I think this is a really great question, and one I don't quite have the answer for yet.

I briefly considered omitting `_` and `$` prefaced properties until we determined if they should be represented differently, but decided against it.

I think leaving them as-is for now to match the AST seems reasonable, but open to other ideas.